mlflow/pyfunc/__init__.py

```diff
@@ -401,8 +401,9 @@ def predict(self, data: PyFuncInput) -> PyFuncOutput:
                      `numpy.ndarray`, `List[numpy.ndarray]`, `Dict[str, numpy.ndarray]` or
                      `pandas.DataFrame`. If data is of `pandas.DataFrame` type and an input field
                      requires multidimensional array input, the corresponding column values in
-                     the pandas DataFrame will be reshaped to the required shape and DataFrame
-                     column values will be cast as the required tensor spec type.
+                     the pandas DataFrame will be reshaped to the required shape with C_CONTIGUOUS
+                     (row-major) order, and DataFrame column values will be cast as the required
+                     tensor spec type.
```

Comment:
What is `C_CONTIGUOUS`?

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.reshape.html?highlight=reshape#numpy.reshape
i.e., reshape(.. , order='C')